### ISSUE TITLE
Help file left with .old extension in en-US dir of the installed module

### ISSUE THREAD

**Issue by rkeithhill** [psake/psake#235]
Help seems to work but I'm not sure how.

**Comment by devblackops** [comment]
PS shouldn’t find the MAML XML so will revert to the comment-based help on all the functions. Now that we have the help in markdown files via PlatyPS, we can either modify the build task to generate the XML help from them or leave it out entirely.

**Comment by rkeithhill** [comment]
Ah.  I thought that platyPS would have ripped out the comment-based help when you ran it to generate the markdown files.  

Don't we want to generate the MAML help file from the markdown files via platyPS?  Otherwise is there any value in using platyPS?  Eventually, I think the PS team wants to replace MAML with markdown in the help system.

**Comment by devblackops** [comment]
Yes, the comment-based help attached to each function should be removed as that will exist in the PlatyPS MD files. We'll need to add a couple of tasks in `build.ps1` to update the MD files as changes are made to the module, and to generate the MAML during the `build` task.

**Comment by HeyItsGilbert** [comment]
This was an artifact of the old in-repo help/build process. Help generation and packaging have since moved to PowerShellBuild, and the current build doesn't leave a .old file behind in en-US. Closing as resolved - if you still hit a stray .old in a fresh install from the gallery, let me know the version and I'll dig in.
